include/seqan3/range/view/translation.hpp

```diff
@@ -295,42 +606,42 @@ namespace seqan3::view
  * translation_frames::WITH_FRAME_SHIFTS        - all forward frames
  * translation_frames::SIX_FRAME                - all forward and reverse frames
  *
- * \par View properties
+ * ### View properties
  *
- * |                     | `irng_t` (range input type)           | `rrng_t` (range return type)                                |
- * |---------------------|---------------------------------------|-------------------------------------------------------------|
- * | range               | seqan3::forward_range_concept         | seqan3::view_concept && seqan3::random_access_range_concept |
- * | `range_reference_t` | seqan3::nucleotide_concept            |                                                             |
+ * |                     | `urng_t` (underlying range type)                                   | `rrng_t` (range return type)                                |
+ * |---------------------|--------------------------------------------------------------------|-------------------------------------------------------------|
+ * | range               | seqan3::sized_range_concept && seqan3::random_access_range_concept | seqan3::view_concept && seqan3::random_access_range_concept |
+ * | `range_reference_t` | seqan3::view_concept                                               | seqan3::aa27                                                |           |
```

Comment:
range_reference_t of the underlying range is not required to be view_concept, instead it is required be nucleotide_concept, or not?
The range_reference_t of the returned range is not aa26, but also random_access_range_concept and view_concept, because the return range is random_access_range of random_access_range, right?

Reply:
yes